app/src/main/java/org/desarrolladorslp/teckersapp/NavigationMenuActivity.kt

```diff
@@ -27,4 +30,20 @@ class NavigationMenu : AppCompatActivity() {
         setupActionBarWithNavController(navController, appBarConfiguration)
         navView.setupWithNavController(navController)
     }
+
+    override fun onClick(v: View) {
+        val i = v.id
+        when (i) {
+            R.id.logoMenu -> logOut()
+        }
+    }
```

Comment:
You could remove the click listener interface from the activity signature and just use an anonymous listener